chat.go

```diff
@@ -129,6 +129,9 @@ func (api *Slack) PostMessage(channelId string, text string, params PostMessageP
 	if params.Username != DEFAULT_MESSAGE_USERNAME {
 		values.Set("username", string(params.Username))
 	}
+	if params.AsUser != false {
```

Comment:
It would be nice to have this false defined as a DEFAULT_MESSAGE_ASUSER. the rest looks good. Thanks for the patch